app/models.py

```diff
@@ -229,3 +238,69 @@ def __repr__(self):
     @classmethod
     def from_name(cls, name):
         return cls.query.filter_by(name=name).first()
+
+
+class BreadOrder(db.Model):
+
+    __tablename__ = 'bread_order'
+
+    id = db.Column(db.Integer, primary_key=True)
+    user_id = db.Column(db.Integer, db.ForeignKey('user.id'))
+    date_id = db.Column(db.Integer, db.ForeignKey('bread_date.id'))
+
+    user = db.relationship('User')
+    bread_date = db.relationship('BreadDate', back_populates="orders")
+    # date = association_proxy('bread_date', 'date')
+
+    items = db.relationship(
+        'BreadType', secondary=association_bread_order_items
+    )
+
+    def __repr__(self):
+        return '<BreadOrder by {} on {}>'.format(
+            self.user.fullname, self.bread_date.date
+        )
+
+    @classmethod
+    def get_all_after_user(cls, start_date, user):
+        """
+        Returns all bread orders after the given start date for a given user.
+        """
+        return (cls.query
+                .join(BreadDate)
+                .filter(cls.user == user)
+                .filter(BreadDate.date > start_date)
+                .all())
```

Comment:
zet dit in queries.py om models.py niet te complex te maken ..
rule of thumb is dat ls je extra models zoals User nodig hebt dat dat in een queries.py komt

Reply:
ik denk dat dit gewoon wegmag eigenlijk.

Reply:
dan mag het weg zeker?